icons/tv-flatscreen.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "ericfennis"
```

Comment:
Are the contributors correct?

Reply:
> Are the contributors correct?

@jguddas No idea 🤷🏻‍♂️😅 I think `contributors` really needs to be something (possibly fully?) automated. Which I think #1343 is tracking…

Reply:
Yes this one is correct, I created the `tv-2` icon originally.